app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Kong jq
+publisher: Kong Inc.
+versions: 0.1.x
+beta: true
+
+desc: Transform JSON objects included in API requests or responses using jq programs.
+description: |
+  The Kong jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
+  The configuration accepts two sets of options: one for the request and another for the response.
```

Comment:
```suggestion
  The Kong jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
  
  The configuration accepts two sets of options: one for the request and another for the response.
```
Adding a line break to separate what the plugin does vs how it works.

Reply:
will update